gluten-flink/docs/Flink.md

```diff
@@ -105,9 +108,35 @@ bin/sql-client.sh -f data-generator.sql
 TODO
 
 ## Performance
-Using the data-generator example, it shows that for native execution, it can generate 10,0000
-records in about 60ms, while Flink generator 10,000 records in about 600ms. It runs 10 times faster.
-More perf cases to be added.
+We are working on supporting the [Nexmark](https://github.com/nexmark/nexmark) benchmark for Flink.
+Now the q0 has been supported.
+
+Results show than running with gluten can be 2.x times faster than Flink.
```

Comment:
👍 

It's a bit counter-intuitive. We can achieve 2x speed up even without batch processing? IIUC the current code is still row-based?

Reply:
We got this result when we use 1024 as a batch for velox. If use one row a batch, velox is much slower than flink. We still need to optimize it.